website/blog/2021-02-22-jest-27.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Jest 27: Leaving the past behind ⏩
+author: Tim Seckinger
+authorURL: https://github.com/jeysal
+authorImageURL: https://avatars.githubusercontent.com/u/16069751
+---
+
+In the [Jest 26 blog post](https://jestjs.io/blog/2020/05/05/jest-26) almost a year ago, we announced that after two major releases with few breaking changes, Jest 27 would flip some switches to set better defaults for projects that are new or can migrate smoothly. This will then also give us the opportunity in Jest 28 to remove some packages from the default distribution of Jest and publish them as separately installable and pluggable modules instead, so that everyone on the new defaults can benefit from a smaller install size.
+
+We will explain those changes of defaults and other notable breaking changes in this post, but first, let's get into some exciting new features shipping in Jest 27!
+
+<!--truncate-->
+
+## Feature updates
+
+Firstly, the interactive mode you may know from reviewing and updating failed snapshots can now also be used to **step through failed tests** one at a time. Credit goes to first-time contributor [@NullDivision](https://github.com/NullDivision) for [implementing](https://github.com/facebook/jest/pull/10858) this feature! ![Interactive failed test run](/img/blog/27-interactive-failures.png)
+
+Speaking of snapshots, one of the more exciting features we've shipped in recent years are Inline Snapshots, which [landed](https://github.com/facebook/jest/pull/6380) in a minor release of Jest 23 almost three years ago. However, they came with the restriction that projects wanting to utilize them must be using [Prettier](https://prettier.io/) to format their code, because that's what Jest would use to make sure the file it writes the snapshots into remains properly formatted.  
```

Comment:
I'm very attached to the way the inline snapshots and perf improvements paragraphs flow, please be nice when reviewing :stuck_out_tongue_closed_eyes: